app/src/main/java/com/brewthings/app/util/Calculations.kt

```diff
@@ -1,20 +1,27 @@
 package com.brewthings.app.util
 
 import com.brewthings.app.data.model.RaptPillData
-import com.brewthings.app.util.datetime.daysBetweenIgnoringTime
 
-fun calculateABV(og: Float, fg: Float, feedings: Float): Float {
-    if (og <= 1.0 || fg <= 1.0) return 0f
-    return (og.plus(feedings) - fg) * 131.25f
+fun calculateFeeding(previousGravity: Float, actualGravity: Float): Float =
+    actualGravity.minus(previousGravity)
```

Comment:
Are you sure the null check is not needed? There are some situations in which it can throw an error.
```suggestion
    if (previousGravity == null) return 0f
    actualGravity.minus(previousGravity)
```